cha/implementaion/playback/decoding.tex

```diff
@@ -0,0 +1,48 @@
+\section{Decoding MP3}
+In \cnameref{cha:initial_development} we presented our initial foundation for development, namely the Google sample implementation of a universal media player app.
+In that implementation an Android implementation, \code{android.media.MediaPlayer}, is used to control the playback of audio streams.
```

Comment:
Implementation Implementation Implementation

Maybe not control but rather "is in charge of playing compressed audio files"